modules/document_repository/php/edit.class.inc

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * This file contains the NDB_Form_media class
+ *
+ * PHP Version 7
+ *
+ * @category Loris
+ * @package  Document_Repository
+ * @author   Shen Wang <wangshen.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/CCNA/
+ */
+namespace LORIS\document_repository;
+
+/**
+ * Creates the form for NDB_Form_media
+ *
+ * @category Loris
+ * @package  Document_Repository
+ * @author   Alex Ilea <ailea.mcin@gmail.com>
+ * @author   Shen Wang <wangshen.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/CCNA/
+ */
+class Edit extends \NDB_Form
+{
+    public $skipTemplate = true;
+
+    /**
+     * Entry point for /media/edit/
```

Comment:
and this one. Basically all references to `media` should be `document_repository`